### ISSUE TITLE
Validation split does not take into account the shuffle=True argument of the tuner search method.

### ISSUE THREAD

**Issue by cevheck** [tensorflow/tensorflow#53211]
https://github.com/tensorflow/tensorflow/blob/80117da12365720167632761a61e0e32e4db2dcc/tensorflow/python/keras/engine/training.py#L1130

I believe I spotted a mistake in the keras tuner code. While shuffle=True is indeed the default argument, it is not taken into account when creating the train and validation set.

Hope this helps for people to come.

**Comment by tilakrayal** [comment]
@cevheck ,
We see that the issue [template](https://github.com/tensorflow/tensorflow/issues/new/choose) has not been filled, could you please do so as it helps us analyse the issue. Can you please elaborate about the statement. Also, please specify the Use Cases for this statement. Thanks!
